scripts/move_to_spark_1.sh

```diff
@@ -2,6 +2,15 @@
 
 set +x
 
+grep -q "spark2" pom.xml
+if [[ $? == 1 ]];
+then
```

Comment:
i think it'd be better, and more consistent w/the other script(s), if you grep for spark1, check for a 0 exit code, and then exit.